nodes/usb_cam_node.cpp

```diff
@@ -87,6 +90,21 @@ class UsbCamNode
     return true;
   }
 
+  bool serviceExposureAuto(std_srvs::SetBool::Request  &req,
```

Comment:
I'm still thinking exposure auto setting would be better to be dynamic_reconfigure.
How do you think?

Reply:
Hmm, I don't know the benefits for using dynamic_reconfigure.
Service is more simple, and it is called just once when robot starts.

Reply:
For example, if we'll also want to update another V4L2 parameters, finally there will be tens of the service servers. dynamic_reconfigure makes it simpler and generic.

Reply:
That makes sense. OK, I'll try it!